sound/effect.h

```diff
@@ -297,7 +297,10 @@ class Effect {
     }
 
     default_output->print("Playing ");
-    default_output->println(filename);
+    default_output->print(filename);
```

Comment:
why are we printing the same filename twice?

Reply:
First print is the wav file including path, second is the effect name, as a debug output to check that the correct effect was selected